pages.es/common/bc.md

```diff
@@ -3,6 +3,14 @@
 > Un lenguaje de calculadora de precisión arbitraria.
 > Más información: <https://manned.org/man/bc.1>.
 
+- Inicia una sesión en modo interactivo
+
+`bc`
+
+- Inicia una sesión en modo interactivo with la biblioteca matemática estándar:
+
+`bc --mathlib`
+
 - Inicia `bc` en el modo interactivo utilizando la biblioteca matemática estándar:
 
 `bc -l`
```

Comment:
```suggestion
- Inicia `bc` en el modo interactivo utilizando la biblioteca matemática estándar:

`bc --mathlib`
```

Mandatory fix: [long option must be used](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#option-syntax).